flatdata-rs/lib/src/storage.rs

```diff
@@ -322,19 +315,13 @@ impl<'a> ResourceHandle<'a> {
 
     /// Close the underlying stream and write the header containing the size in
     /// bytes of written data.
-    pub fn close(&mut self) -> Result<&'a [u8], ResourceStorageError> {
+    pub fn close(self) -> Result<&'a [u8], ResourceStorageError> {
```

Comment:
This is nice, but I don't get to what 'a is bound now.

Reply:
'a is a parameter of the class (since the memory is owned by the storage)